lib/unison-sqlite/src/Unison/Sqlite/Connection.hs

```diff
@@ -176,18 +211,43 @@ queryListRow conn@(Connection _ _ conn0) s params =
           sql = s
         }
   where
-    doQueryListRow = do
-      debugM Debug.Sqlite "query" s
-      debugM Debug.Sqlite "params" (anythingToString params)
-      result <- run
-      debugM Debug.Sqlite "result" (anythingToString params)
-      debugLogM Debug.Sqlite "----------"
-      pure result
-    run = Sqlite.query conn0 (coerce s) params
+    thing =
```

Comment:
```suggestion
    withStatement =
```

Reply:
I went with `run`; it's not really a statement that it's running.